resources/assets/coffee/react/mp-history/main.coffee

```diff
@@ -52,8 +53,17 @@ class MPHistory.Main extends React.Component
     .done (data) =>
       return if _.isEmpty data.events.data
 
-      newEvents = _.concat @state.events, data.events.data
-      lastEvent = _.last(newEvents)
+      lastIndex = _.findLastIndex @state.events, (o) => o.id == @state.since + 1
```

Comment:
so that we can remove the event with a match in progress (+ any other event that was created after it) and replace it with ones we got in an update